spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/config/KinesisBinderConfiguration.java

```diff
@@ -93,7 +92,7 @@ public KinesisMessageChannelBinder kinesisMessageChannelBinder(
 			KinesisExtendedBindingProperties kinesisExtendedBindingProperties,
 			@Autowired(required = false) KinesisProducerConfiguration kinesisProducerConfiguration,
 			AWSCredentialsProvider awsCredentialsProvider,
-			@Autowired(required = false) @Qualifier("kclTaskExecutor") TaskExecutor kclTaskExecutor) {
+			@Autowired(required = false) @Qualifier("taskScheduler") TaskExecutor kclTaskExecutor) {
```

Comment:
No, my point is do not inject any `TaskExecutor` at all and just rely on the default one created in the channel adapter.